crates/masonry/src/testing/harness.rs

```diff
@@ -175,11 +175,24 @@ impl TestHarness {
         let window_size = PhysicalSize::new(window_size.width as _, window_size.height as _);
 
         let mut harness = TestHarness {
-            render_root: RenderRoot::new(root_widget, WindowSizePolicy::User),
+            render_root: RenderRoot::new(
+                root_widget,
+                RenderRootOptions {
+                    use_system_fonts: false,
+                    size_policy: WindowSizePolicy::User,
+                },
+            ),
             mouse_state,
             window_size,
             background_color,
         };
+        const FONT_PATH: &str = concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/resources/fonts/roboto/Roboto-Regular.ttf"
+        );
```

Comment:
Wouldn't that path be wrong for any dependent crate creating a test harness?

I think you want to use `include_bytes` instead (until we come up with an actual resource loading system).